Assets/Source/Options.cs

```diff
@@ -4,13 +4,36 @@
 namespace HiveMind
 {
 	public class Options : MonoBehaviour {
```

Comment:
@leyarotheconquerer Can you explain your reasoning behind this change? I'm not sure what advantage this gives us.